examples/typescript-kubernetes/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "typescript-kubernetes",
+  "version": "1.0.0",
+  "main": "main.js",
+  "types": "main.ts",
+  "license": "MPL-2.0",
+  "private": true,
+  "scripts": {
+    "get": "cdktf get",
+    "build": "yarn get && tsc",
+    "synth": "cdktf synth",
+    "compile": "tsc --pretty",
+    "watch": "tsc -w",
+    "test": "echo ok",
+    "upgrade": "npm i cdktf@latest cdktf-cli@latest",
+    "upgrade:next": "npm i cdktf@next cdktf-cli@next"
+  },
+  "engines": {
+    "node": ">=10.12"
+  },
+  "dependencies": {
+    "cdktf": "0.0.0",
+    "constructs": "^3.0.4"
```

Comment:
This has to be `"constructs": "^3.0.0"`, since it's a mono repo and otherwise we'd have version conflicts and therefore typescript errors.